spec/raven/event_spec.rb

```diff
@@ -347,6 +347,7 @@ def raven_context
   end
 
   describe '.initialize' do
+    # NOTE: This test is completely pointless.
```

Comment:
The expectation is that the code will not raise an exception. You can look back through the `git blame` for why.